library/redhat_subscription_el7.py

```
@@ -0,0 +1,1295 @@
+    """
+
+    def __init__(self, module, **kwargs):
+        self.module = module
+        for k, v in kwargs.items():
+            setattr(self, k, v)
+
+    def __str__(self):
+        return str(self.__getattribute__("_name"))
+
+    def get_pool_id(self):
+        return getattr(self, "PoolId", getattr(self, "PoolID"))
```

Ruff (0.16.1)

[warning] Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)